ggml/src/ggml-cpu/ops.cpp

```diff
@@ -3963,7 +3965,10 @@ static void ggml_compute_forward_scale_f32(
             // src0 is same shape as dst => same indices
             memcpy((char *)dst->data + i1*nb1, (char *)src0->data + i1*nb01, nc * sizeof(float));
         }
-        ggml_vec_scale_f32(nc, (float *) ((char *) dst->data + i1*nb1), v);
+        ggml_vec_scale_f32(nc, (float *) ((char *) dst->data + i1*nb1), s);
+        if (b != 0.0f) {
+            ggml_vec_acc1_f32(nc, (float *) ((char *) dst->data + i1*nb1), b);
+        }
```

Comment:
Merge these in `ggml_vec_mad1_f32()`. If you want, you can try to add a `GGML_SIMD` version using `GGML_F32_VEC_FMA` - it's quite simple. But also can leave it a basic `for` loop without SIMD.